src/App.tsx

```diff
@@ -747,7 +751,11 @@ function App() {
                         (account: CaipAccountId) => {
                           const { address } = parseCaipAccountId(account);
                           return (
-                            <option key={address} value={account}>
+                            <option
+                              data-testid={`${account}`}
```

Comment:
These are unique because they are CAIP-10 addresses (meaning they contain the scope in the address as well?)